lib/views/github-login-view.js

```diff
@@ -63,8 +63,12 @@ export default class GithubLoginView extends React.Component {
         <div className="github-GitHub-LargeIcon icon icon-mark-github" />
         <h1>Enter Token</h1>
         <ol>
-          <li>Visit <a href="https://github.atom.io/login">github.atom.io/login</a> to generate
+          <li>Visit <a href="https://github.com/settings/tokens">github.com/settings/tokens</a> to generate
           an authentication token.</li>
+          <ul>
+            <li>Generate a new token (classic) with at least the following permissions:
+            <code>repo</code>, <code>workflow</code>, <code>read:org</code>, and <code>user:email</code>.</li>
+          </ul>
```

Comment:
```suggestion
          <li>
            Visit <a href="https://github.com/settings/tokens">GitHub</a> to generate a new
            <a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic"> Personal
            Access Token (classic)</a>.
          </li>
          <li> 
            Ensure it has the following permissions <code>repo</code>, <code>workflow</code>, <code>read:org</code>, and <code>user:email</code>.
          </li>
          <li>
            Enter the token below:
          </li>
```

I wonder if this could be made a bit more concise by not repeating instructions ("generate token" is mentioned twice) and could be more helpful with a link to the GH docs on how to actually create one.
I also feel the `<ul>` is a bit weird as it feels like a separate step but it isn't nested with the previous item and doesn't have a marker.

The above suggestion would look something like:
![image](https://user-images.githubusercontent.com/58074586/219989392-931a9ecb-e28f-4045-8c42-e486ea753c63.png)

Reply:
I liked the simplified wording, thank you for that.

Also liked some of the HTML refactoring to be slightly neater, but in terms of whitespace-only suggested changes, I decided to stay with same-line/end-of-line opening/closing tags in some places, both for visual conciseness and to match surrounding HTML (JSX) code style. If we wanna refactor the use of whitespace and indentation, then I'd like it to be mostly consistent throughout the file.

I felt that the link to make the token being visually smaller than the documentation link was a no-go for user experience (UX) reasons, so I moved the docs link into its own `<sup></sup` element to make it look like a "footnote" or "further reading" link, and restored suggested "GitHub" back to "github.com/settings/tokens" -- I like that this shows you exactly where on github.com you need to go, in case you don't realize you can click through the link or your default URL handler/browser is badly configured (??), but also to make this link _visually dominate_ (by virtue of being much larger than the "<sup>[docs]</sup>" link), since that's where we want people to go, and I don't want to "make people think" in the immortal words of Steve Apple, may he rest in peace.

Screenshot:

![github package login view showing instructions and a token entry field, Cancel and Login buttons](https://user-images.githubusercontent.com/20157115/222232221-7c76619b-61e8-4d7b-b129-dece3c22bca8.png)

Diff:

This comment is discussing changes I committed to this PR's branch in commit https://github.com/pulsar-edit/github/pull/15/commits/249a9862ab07f152827f9f3d84b4efadf0db3bac.

Reply:
I think that is a good hybrid of both, seems good to me.